Views/Home/Index.cshtml

```diff
@@ -1,91 +1,1358 @@
   <title>Musik | Web Application</title>
   <meta name="description" content="app, web app, responsive, admin dashboard, admin, flat, flat ui, ui kit, off screen nav" />
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
-  <link rel="stylesheet" href="js/jPlayer/jplayer.flat.css" type="text/css" />
-  <link rel="stylesheet" href="css/bootstrap.css" type="text/css" />
-  <link rel="stylesheet" href="css/animate.css" type="text/css" />
-  <link rel="stylesheet" href="css/font-awesome.min.css" type="text/css" />
-  <link rel="stylesheet" href="css/simple-line-icons.css" type="text/css" />
-  <link rel="stylesheet" href="css/font.css" type="text/css" />
-  <link rel="stylesheet" href="css/app.css" type="text/css" />  
-  <script src="https://apis.google.com/js/platform.js" async defer></script>
+ <link rel="stylesheet" href="/css2/js/jPlayer/jplayer.flat.css" type="text/css" />
+  <link rel="stylesheet" href="/css2/bootstrap.css" type="text/css" />
+  <link rel="stylesheet" href="/css2/animate.css" type="text/css" />
+  <link rel="stylesheet" href="/css2/font-awesome.min.css" type="text/css" />
+  <link rel="stylesheet" href="/css2/simple-line-icons.css" type="text/css" />
+  <link rel="stylesheet" href="/css2/font.css" type="text/css" />
+  <link rel="stylesheet" href="/css2/app.css" type="text/css" />  
     <!--[if lt IE 9]>
     <script src="js/ie/html5shiv.js"></script>
     <script src="js/ie/respond.min.js"></script>
     <script src="js/ie/excanvas.js"></script>
   <![endif]-->
 </head>
-<body class="bg-info dker">
-  <section id="content" class="m-t-lg wrapper-md animated fadeInDown">
-    <div class="container aside-xl">
-      <section class="m-b-lg">
-        <header class="wrapper text-center">
-          <strong>Sign in to find interesting thing</strong>
-        </header>
-@using (Html.BeginForm("Verify", "Home", FormMethod.Post, new { @class = "form-horizontal", role = "form" }))
-{
-
-    @Html.TextBoxFor(c =>c.ConsumerModel.Email,new{@Name="Email", @type="email", @class="form-control",@placeholder="Email",@autocomplete="off" , @required="required"})
-   @Html.TextBoxFor(c =>c.ConsumerModel.Pass_word,new{@Name="Pass_word", @type="password", @class="form-control",@placeholder="Password",@autocomplete="off" , @required="required"})
-   
-    <div class="form-group">
-        <div class="col-md-offset-2 col-md-10">
-            <input type="submit" value="Submit" class="btn btn-default" />
+<body class="">
+  <section class="vbox">
+    <header class="bg-white-only header header-md navbar navbar-fixed-top-xs">
+      <div class="navbar-header aside bg-info nav-xs">
+        <a class="btn btn-link visible-xs" data-toggle="class:nav-off-screen,open" data-target="#nav,html">
+          <i class="icon-list"></i>
+        </a>
+        <a href="index.html" class="navbar-brand text-lt">
+          <i class="icon-earphones"></i>
+          <img src="/css2/images/logo.png" alt="." class="hide">
```

Comment:
I mentioned them on the video but I’ll write it here too.

1) The logo on the top left
2) Logout button
3) Positioning of the music bar
4) Positioning of the picture on the left panel